scripts/version-updater.gradle

```diff
@@ -6,13 +6,19 @@ ext {
     gradlePropertiesFilePath = "${rootProject.projectDir}/version.properties"
     gliaCoreSdkVersionTagInProperties = 'dependency.coreSdk.version'
     gliaCoreSdkVersionTag = 'gliaSdkVersion'
+    wdigetsVersionNameTagInProperties = 'widgets.versionName'
+    wdigetsVersionNameTag = 'widgetsVersionName'
```

Comment:
Here and in other places, `wdigetsVersionNameTag` => `widgetsVersionNameTag` (minor typo)